networking/hardware_networks/configuring-sriov-ib-attach.adoc

```diff
@@ -19,6 +19,9 @@ include::modules/nw-multus-configure-dualstack-ip-address.adoc[leveloffset=+2]
 // Configuration of IP address assignment for a network attachment
 include::modules/nw-multus-ipam-object.adoc[leveloffset=+2]
 
+// Dynamic IP address assignment configuration with Whereabouts
+include::modules/nw-multus-whereabouts.adoc[leveloffset=+3]
```

Comment:
might want to revisit the heading level based on the correction above. Same applies to other instances as well